jina/docker/helper.py

```diff
@@ -36,12 +37,12 @@ def get_default_login():
     return login_info
 
 
-def handle_dot_in_keys(document):
+def handle_dot_in_keys(document: Dict[str, Union[Dict, List]]) -> Union[Dict, List]:
     updated_document = {}
     for key, value in document.items():
-        if isinstance(value, dict):
+        if isinstance(value, Dict):
```

Comment:
let's keep it as it was, type hinting is not enforced by Python, and I am not sure how interchangable in terms of functionality `Dict` vs `dict` is. These PRs should only affect function signatures' type hinting but not touch the function.

Reply:
this is wrong, use `dict`

Reply:
Done